c/registerProduct.c

```diff
@@ -93,47 +94,49 @@ int productRegistration(const char *major_version,
 
   /* Register Product with IFAUSAGE */
 
-  arg = (IFAARGS_t *)__malloc31(sizeof(IFAARGS_t));
+  /* arg = (IFAARGS_t *)__malloc31(sizeof(IFAARGS_t)); */
```

Comment:
Dead code.

Reply:
Will remove

Reply:
Thank you Irek, The code is much cleaner now.  Thanks again.